clang/lib/Driver/ToolChains/WebAssembly.cpp

```diff
@@ -572,6 +572,7 @@ void cheerp::CheerpOptimizer::ConstructJob(Compilation &C, const JobAction &JA,
     CmdArgs.push_back("-FreeAndDeleteRemoval");
     CmdArgs.push_back("-cheerp-lto");
     CmdArgs.push_back("-Os");
+    CmdArgs.push_back("-PartialExecuter");
```

Comment:
This commit should be ordered _after_ the pass is actually introduced

Reply:
ok